webapp/src/components/CreateIouDialog.tsx

```diff
@@ -118,9 +116,7 @@ export const CreateIouDialog: React.FC<{
                             variant="outlined"
                             value={payee}
                             type={'email'}
-                            onChange={(e) =>
```

Comment:
Can we use Sonar or at least github actions for linters

Reply:
Yes, please :D 
https://noumenadigital.atlassian.net/browse/ST-4191